### PR TITLE
fix: add missed IBM MQ Operation Binding

### PR DESCRIPTION
**Pull request by magicmatatjahu**
I found that we have forgot to define (probably it's a bug) definition for `IBM MQ Operation Binding`. I know that some bindings don't need to have definition for particular kind of binding, but we should have reserved keywords.

I found it in review of https://github.com/asyncapi/spec/pull/836 PR 

cc @derberg @fmvilas and @dalelane you should be most interested :)

### CODE REVIEW

**Comment by char0n** [comment]
@magicmatatjahu referencing also previous discussion about this in https://github.com/asyncapi/spec/issues/605

**Comment by magicmatatjahu** [comment]
@char0n Hah, thanks! I didn't see your issue. However @derberg add comment https://github.com/asyncapi/spec/pull/836#issuecomment-1252443273 so I don't know how I should think about this inconsistency across bindings.

**Comment by fmvilas** [comment]
LGTM but I don't see an "Operation Binding Object" section on [the `ibmmq` binding](https://github.com/asyncapi/bindings/tree/master/ibmmq). We should also add it there and specify that it's just empty and reserved for future use ([see example](https://github.com/asyncapi/bindings/tree/master/amqp1#operation-binding-object)).

**Comment by derberg** [comment]
yeah, I think we should make sure ibm mq is there too, for consistency reasons

**Review by dalelane — APPROVED** [review]

**Comment by magicmatatjahu** [comment]
PR https://github.com/asyncapi/bindings/pull/149 is merged. It can be merged too :)

cc @char0n

**Comment by char0n** [comment]
@fmvilas @derberg can you please look into this PR? THanks!

**Comment by char0n** [comment]
@magicmatatjahu is there any adaption required for `parser-js` after this PR is merged?

**Comment by magicmatatjahu** [comment]
Nope, ParserJS treats bindings atm as `Map<string, any>` values.

**Review by derberg — APPROVED** [review]
oh nice, https://github.com/asyncapi/bindings/pull/149 is merged

lets get it in

**Comment by derberg** [comment]
@char0n as Fran wrote ` Approved it but let's wait for @rcoppen and others to review it.`, IMHO you can merge as others did review and @rcoppen is aware as he had to approve the change in the binding to get it marget, as he is a codeowner of the IBMMQ binding

**Comment by fmvilas** [comment]
Yup, feel free to merge.

**Pull request merged**
